src/styles/blocks/about.scss

```
@@ -0,0 +1,109 @@
+.about {
+  display: flex;
+  gap: 110px;
+  align-items: center;
+  position: relative;
+  padding: 0 0 200px 80px;
+  justify-content: flex-start;
+  @media (max-width: 768px) {
```

```suggestion
  justify-content: flex-start;
  
  @media (max-width: 768px) {
```
For better reading add spaces